actions/pyproject.toml

```diff
@@ -2,20 +2,19 @@
 name = "robocorp-actions"
 version = "0.0.2"
 description = "Robocorp Actions"
-authors = [
-	"Fabio Zadrozny <fabiofz@gmail.com>",
-]
+authors = ["Fabio Zadrozny <fabiofz@gmail.com>"]
 readme = "README.md"
 repository = "https://github.com/robocorp/robo/"
 license = "Apache-2.0"
-packages = [{include = "robocorp", from="src"}]
+packages = [{ include = "robocorp", from = "src" }]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-robocorp-tasks = "^2.6.0"
+# robocorp-tasks = "^2.6.0"
+robocorp-tasks = { path = "../tasks", develop = true }
```

Comment:
wip: release new package version and replace

Reply:
@fabioz is there a simple way to always use the local packages when developing locally and versioned ones only when releasing?

Reply:
Yes: `inv devinstall` (please revert these changes).